compiler/src/iree/compiler/Preprocessing/TransformExtensions/PreprocessingExtensionsOps.td

```diff
@@ -219,6 +219,7 @@ def MatchContractionOp : Op<Transform_Dialect, "iree.match.contraction",
     `,` `lhs_type` `=` $lhs_type
     `,` `rhs_type` `=` $rhs_type
     `,` `output_type` `=` $output_type
+    (`,` `indexing_maps` `=` $indexing_maps^)?
```

Comment:
Can we also use this syntax in all tests and add an op example in `let description = ` above?